backend/commands/import_attendance_list.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import click
+
+from flask.cli import with_appcontext
+from backend.models import Hacknight, Participant
+from backend.extensions import db
+
+
+@click.command()
+@with_appcontext
+def import_attendance_list():
+    """Import attendance list."""
+    rows = []
+    list_of_participants = []
+    users_dict = {}
+
+    # open file from exported csv
+    with open("./temp/list.csv") as csv_file:
+        csv_reader = csv.reader(csv_file, delimiter=",")
+        for row in csv_reader:
+            rows.append(row)
+
+    # get dates of historical hacknights
+    dates = rows[0][4:]
+    click.echo(f"Created list of dates: {dates}")
+
+    # skip header
+    rows = rows[1:]
+    # prepare users' data
+    for item in rows:
+        _id, github, name, phone, email, *values = item
```

Comment:
```suggestion
        _id, github, name, phone, email, *_ = item
```

*incredible nitpick* :P